client/plots/disco/launch.adhoc.ts

```diff
@@ -1,20 +1,29 @@
 import { Genome } from '#shared/types/index'
 
-/**
-from multiple possible data sources, parse all into an array and return
-app drawer will supply data for "snvText, svText, cnvText"
-while the "File" and "Url" can come from url parameters e.g. host?disco=1&snvFile=path/to/file
-
-Move these types to shared/server if used across multiple files
-
-*/
 type MutationListEntry = {
-	/**json list of data points, each element is an event with "dt:int" and ready to be used by disco */
+	/**json list of alteration events, each element is an event with "dt:int" and ready to be used by disco
+	FIXME different dt elements require different attributes but this design merge all into one type, can improve later
```

Comment:
should change to define 3 separate types for `SnvType, CnvType, SvType`. how to indicate the mlst[] can contain all these types?

Reply:
will do an union type